.pep8speaks.yml

```diff
@@ -1,12 +0,0 @@
-scanner:
```

Comment:
I know it is a duplicate but I I prefer to keep pep8speaks for now. 
people react more and faster on a new comment than a failed CI.

Reply:
OK. Will keep it in the new push force, but we may run into conflicts if slight differences exist in the way `ruff` enforces some rules. We'll keep an eye on this.

Reply:
Restored it in 20e1a17.